src/go/rpk/pkg/cli/cmd/cluster/license/info.go

```diff
@@ -0,0 +1,68 @@
+package license
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newInfoCommand(fs afero.Fs) *cobra.Command {
+	var jsonFormat bool
+	command := &cobra.Command{
+		Use:   "info",
+		Args:  cobra.ExactArgs(0),
+		Short: "Retrieve license information",
+		Long: `Retrieve license information:
+
+    Organization:    Organization the license was generated for.
+    Type:            Type of license: free, enterprise, etc.
+    Expires:         Number of days the license is valid until, -1 if is expired.
+    Version:         License schema version.
+`,
+		Run: func(cmd *cobra.Command, args []string) {
+			p := config.ParamsFromCommand(cmd)
+			cfg, err := p.Load(fs)
+			out.MaybeDie(err, "unable to load config: %v", err)
+
+			cl, err := admin.NewClient(fs, cfg)
+			out.MaybeDie(err, "unable to initialize admin client: %v", err)
+
+			info, err := cl.GetLicenseInfo(cmd.Context())
+			out.MaybeDie(err, "unable to retrieve license info: %v", err)
+
+			if !info.Loaded {
+				out.Die("license is not loaded for this cluster")
```

Comment:
Since this is a more proper exit, I think we may want a realer sentence here:
```
This cluster is missing a license.
```
?

Reply:
I'll change that but lowercase and without a punctuation mark since we are sending it to stderr.